Sources/Todo.Services/TodoItemQuery.cs

```diff
@@ -13,5 +14,30 @@ public class TodoItemQuery
 
         [Required]
         public ClaimsPrincipal User { get; set; }
+
+        [Range(1, 1000)]
+        public int PageSize { get; set; } = 25;
+
+        [Range(0, int.MaxValue)]
+        public int PageIndex { get; set; } = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'PageIndex', the compiler will do that for you.](https://app.codacy.com/manual/satrapu/aspnet-core-logging/pullRequest?prid=5148460)